goaviatrix/tags.go

```diff
@@ -91,6 +104,32 @@ func (c *Client) GetTags(tags *Tags) ([]string, error) {
 	return nil, nil
 }
 
+func (c *Client) GetTagsMap(tags *Tags) error {
```

Comment:
Again, can we avoid methods that only mutate an objects state? Instead just return the tags and leave the passed in object unmodified